src/dom/pure.ts

```diff
@@ -1,4 +1,4 @@
-import ReactDOM from 'react-dom'
+import * as ReactDOM from 'react-dom'
```

Comment:
This prevents the import relying on `@babel/runtime` for default import interop.

Reply:
so this works the same either way? interesting

Reply:
It's interesting because `import ReactDOM` and `import * as ReactDOM` are _not_ equivalent as per the ES6 import spec, but most compilers/bundlers treat them interchangeably.  Babel does this by inserting the default import interop helper when there is no default export for the module, like in [React](https://github.com/facebook/react/blob/master/packages/react/index.js)/[ReactDOM](https://github.com/facebook/react/blob/master/packages/react-dom/index.js)'s case.  By changing the the `import * as ...`, it's actually _more_ correct for these libraries.